### ISSUE TITLE
Enable torch.nn.modules.normalization typechecks during CI

### ISSUE THREAD

**Issue by guilhermeleobas** [pytorch/pytorch#49034]
`TestTypeHints.test_run_mypy` should pass with `[mypy-torch.nn.modules.normalization]` deleted from mypy.ini

cc @ezyang @malfet @rgommers @xuzhao9 @gramster
